app/api-v1/routes/order.js

```diff
@@ -6,13 +6,21 @@ module.exports = function (orderService, identityService) {
       res.status(500).json({ message: 'Not Implemented' })
     },
     POST: async function (req, res) {
-      const { address: supplierAddress } = await identityService.getMemberByAlias(req, req.body.supplier)
-
-      const { statusCode, result } = await orderService.postOrder({ ...req.body, supplier: supplierAddress })
       if (!req.body) {
         throw new BadRequestError({ message: 'No body uploaded', service: 'order' })
       }
-      return res.status(statusCode).json({ ...result[0], supplier: req.body.supplier })
+
+      const response = await identityService.getMemberByAlias(req, req.body.supplier)
+
+      const selfAddress = await identityService.getMemberBySelf(req, response.address)
+
+      const { statusCode, result } = await orderService.postOrder({
+        ...req.body,
+        supplier: response.address,
```

Comment:
maybe a different naming? the response should be what we returning back in this context. Why not to call it memberAddress? or just a `member`?